### PR TITLE
add sentry monitoring

### PR DESCRIPTION
**Pull request by spiderxm**
## Description
Add sentry monitoring

## Motivation and Context
Sentry helps us to monitor, debug bugs is our code by alerting and providing various details. Example given below - 
I turned off local elasticsearch server and sentry notified me about the bug:

## Screenshots 
Screenshot 2022-07-11 at 10 55 12 PM
Screenshot 2022-07-11 at 10 55 23 PM

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other

#### Checklist
- [x] I have followed the [Contributor Guidelines](https://github.com/OWASP/Nettacker/wiki/Developers#contribution-guidelines).
- [ x The code has been thoroughly tested in my local development environment with flake8 and pylint.
- [x] The code is Python 3 compatible.
- [x] The code follows the PEP8 styling guidelines with 4 spaces indentation.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -105,3 +105,18 @@ def user_configuration():
...
         "default_excluded_modules": None  # or any module name separated with comma
     }
+
+
+def sentry_configuration() -> dict:
+    """
+        sentry configuration
+
+    Returns:
+        JSON/Dict sentry configuration
```
Comment (dhirensr) [inline config.py:115]: this would need to be documented in the wiki too I believe after merging

File: config.py [hunk 2]
```diff
@@ -105,3 +105,18 @@ def user_configuration():
...
+    """
+    return {
+        "sentry_monitoring": False,
+        "sentry_dsn_url": "",
+        # Enter your Sentry Project URL here,
```
Comment (dhirensr) [inline config.py:120]: can you comment it above please?

File: config.py [hunk 3]
```diff
@@ -105,3 +105,18 @@ def user_configuration():
...
+        "sentry_trace_rate": 1  # Enter Trace Rate Here.
```
Comment (dhirensr) [inline config.py:121]: can you also explain in comment what trace rate would do?

File: config.py (hunk 1 above)
Comment (spiderxm) [reply config.py:115]: yes, will add proper documentation in wiki

**Review by dhirensr — APPROVED** [review]

**Pull request merged**